deploy/determined_deploy/aws/cli.py

```diff
@@ -163,7 +185,7 @@ def deploy_aws(args: argparse.Namespace) -> None:
 
     det_configs = {
         constants.cloudformation.KEYPAIR: args.keypair,
-        constants.cloudformation.ENABLE_CORS: args.enable_cors,
+        constants.cloudformation.ENABLE_CORS: args.enable_cors,  # TODO: Has this been tested? Based on spot handling of bools, I wouldn't expect this to work correctly...
```

Comment:
@hamidzr you verified this correct?

Reply:
Sorry, that comment is out-of-date - this wasn't a real problem.

Reply:
Hmm I tested with det-deploy aws up. Would it cover spot instances. My comment.might be out of date at this point

Reply:
Ignore this. I had an issue during testing and I thought it was because when booleans get injected into the master.yaml, they have a capital letter (`True`/`False`). But I forgot that is a perfectly valid boolean in YAML.